src/components/minor-button/minor-button.stories.mdx

```diff
@@ -0,0 +1,417 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs";
+import Button from "../button";
+import MinorButton from ".";
+import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
+
+<Meta title="Minor Button" parameters={{ info: { disable: true } }} />
+
+# Minor Button
+
+A Button triggers a single action or event.
+Use it to submit a form (Save), to advance to the next step in a process (Next), or to create a new item (New).
```

Comment:
What do you think about amending the descriptions like this one to explain why `MinorButton` is different to `Button` (aka button major), similarly to how Design System have done in their documentation? 
I think this would help consumers understand why we have two separate button components and where each type should be used.

Perhaps @LaiaSage may be able to help you with this? 😄

Reply:
Please keep in mind that  PR which is about separating Designer Notes from Carbon is in the pipeline too. I think we would require more Developer's comments for that one? What do you think @Parsium ?